yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -155,6 +155,29 @@ func saveMigrationReportsFn(msr *metadb.MigrationStatusRecord) {
 		}
 	}
 
+	askAndStorePasswords(msr)
+	passwordsEnvVars := []string{
+		fmt.Sprintf("TARGET_DB_PASSWORD=%s", targetDBPassword),
+		fmt.Sprintf("FF_DB_PASSWORD=%s", fallForwardDBPassword),
+		fmt.Sprintf("SOURCE_DB_PASSWORD=%s", sourceDBPassword),
+	}
+
+	if msr.ExportType == "snapshot-and-changes" { // streaming changes case
+		utils.PrintAndLog("save live migration reports...")
+		liveMigrationReportFilePath := filepath.Join(backupDir, "reports", "live_migration_report.txt")
```

Comment:
let's call it "data migration report" instead of "live migration report"? As we've decided to change the command name with the intention of also support offline migration report in the future.